gen

```diff
@@ -104,7 +104,7 @@ Generate FILE, auto-input typical text, and auto-launch default editor.
 Full documentation <https://github.com/membersincewayback/gen>
 ${version}
 EOF
-exit
+exit $1
```

Comment:
Why $1?

Reply:
Nevermind, makes sense. It does need to be quoted however.

Reply:
I'm not sure I understand, why should it be quoted?

Reply:
Must pass shellcheck. I know without checking "$1" will pass and yours will not.

Reply:
Oooh, okay lol  
[c2a9f7a](https://github.com/membersincewayback/gen/pull/15/commits/c2a9f7a) fixes this